homeassistant/components/mqtt/config_flow.py

```diff
@@ -620,6 +627,44 @@
     ),
     vol.Coerce(int),
 )
+IMAGE_CONTENT_TYPE_SELECTOR = SelectSelector(
+    SelectSelectorConfig(
+        options=[
+            SelectOptionDict(
+                value="image/jpeg", label="Joint Photographic Expert Group image (JPEG)"
+            ),
+            SelectOptionDict(
+                value="image/png", label="Portable Network Graphics (PNG)"
+            ),
+            SelectOptionDict(
+                value="image/apng", label="Animated Portable Network Graphics (APNG)"
+            ),
+            SelectOptionDict(value="image/avif", label="AV1 Image File Format (AVIF)"),
+            SelectOptionDict(
+                value="image/gif", label="Graphics Interchange Format (GIF)"
+            ),
+            SelectOptionDict(
+                value="image/svg+xml", label="Scalable Vector Graphics (SVG)"
+            ),
+            SelectOptionDict(value="image/webp", label="Web Picture format (WEBP)"),
+        ],
+        mode=SelectSelectorMode.DROPDOWN,
+    )
+)
+IMAGE_ENCODING_SELECTOR = SelectSelector(
+    SelectSelectorConfig(
+        options=[
+            SelectOptionDict(value="b64", label="Base64"),
+        ],
+        mode=SelectSelectorMode.DROPDOWN,
+    )
+)
```

Comment:
I get this is optional because there's either no encoding (raw bytes) or base64-encoding.
How does this appear to the user though? Please share some screenshots.

Reply:
<img width="562" height="671" alt="afbeelding" src="https://github.com/user-attachments/assets/b0b2269c-2596-4ddc-9a5f-73a6638c9f79" />

<img width="545" height="692" alt="afbeelding" src="https://github.com/user-attachments/assets/167c5560-51fc-46fd-9584-6521a2f4684b" />

Reply:
Have added an option for Raw data:

<img width="462" height="594" alt="afbeelding" src="https://github.com/user-attachments/assets/29eb7951-6433-4962-83f5-e3778f5de37f" />